docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/NetworkBuilder.java

```diff
@@ -82,9 +82,9 @@ public NetworkBuilder addIpam(Map<String, Object> ipamConfig) {
     private List<IPAMConfig> createIpamConfig(Collection<Map<String, Object>> configs) {
         List<IPAMConfig> ipamConfigs = new ArrayList<>();
         for (Object config : configs) {
-            if (config instanceof LinkedHashMap) {
+            if (config instanceof Map) {
```

Comment:
Each element in the collection is a `Map` as per method signature. There is no need for using `instanceof` and casting if you tweak for-each loop properly.